logs_rollbar.module

```diff
@@ -377,3 +379,26 @@ function logs_rollbar_get_env_uuid() {
   return $server_uuid;
 }
 
+/**
+ * Returns a list of comma separated clientside messages, for Rollbar to ignore.
+ *
+ * @return string
+ *   Formatted list of comma separated messages, Rollbar will ignore.
+ */
+function logs_rollbar_get_ignored_clientside_messages() {
+  $ignored_messages = variable_get('logs_rollbar_ignored_clientside_messages', '');
+
+  $messages_to_ignore = array();
+  foreach (explode(',', $ignored_messages) as $message) {
+    $message = trim($message);
+
+    if ($length = strlen($message) === 0) {
+      continue;
+    }
+
+    $message = $message[0] == '\'' ? $message : '\'' . $message;
```

Comment:
we need to escape the `'` character, see this example from a production site:
![image](https://user-images.githubusercontent.com/114076/39566827-29622b80-4ebd-11e8-98a4-e270807c36cd.png)

Reply:
@AronNovak 
Is this better:

![selection_999 1078](https://user-images.githubusercontent.com/17882245/39572548-2cbf8dac-4ed8-11e8-8ee3-7bffecac1e8a.png)

Reply:
It is, on this screenshot, it seems that the escaping is just fine.